src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
Yes, sorry I wasn't clear. My point was that you're wrapping `DetectRTC` here.

I'm suggesting that you just call `DetectRTC.isMobileDevice` directly, rather than using this `isDesktop` helper to do it.

Or, if you want to wrap `DetectRTC`, then entirely wrap it (so move all references to it, to helpers like this one instead)